### PR TITLE
Add IBM MQ to the Ballerina Library Dashboard

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

File: release/resources/stdlib_modules.json [hunk 1]
```diff
@@ -556,6 +556,16 @@
...
+        {
+            "name": "module-ballerinax-ibm.ibmmq",
+            "module_version": "0.1.4-SNAPSHOT",
+            "level": 1,
+            "default_branch": "main",
+            "version_key": "stdlibIbmMqVersion",
+            "release": true,
+            "dependents": [],
+            "display_code_cov_badge": true
+        },
```
Comment (NipunaRanasinghe) [inline release/resources/stdlib_modules.json:568]: do we need this entry? IINM we only need to update the `module_list.json` and the rest should get auto-generated (@ThisaruGuruge correct me if um wrong)

Comment (ayeshLK) [reply release/resources/stdlib_modules.json:568]: I saw that several other connectors already followed the same convention.

Comment (ThisaruGuruge) [reply release/resources/stdlib_modules.json:568]: @NipunaRanasinghe is correct. We don't need this entry, as mentioned in the [Adding a Module](https://github.com/ballerina-platform/ballerina-library/blob/main/docs/adding-a-new-ballerina-module.md#add-the-module-to-the-ballerina-library-optional) doc. 

@ayeshLK Yes, I have mentioned this multiple times in the past, but still people not following it.

Comment (ayeshLK) [reply release/resources/stdlib_modules.json]: Fixed with [0d5a33887f06ad77aa8f607f51da628f6c7056e8](https://github.com/ballerina-platform/ballerina-library/pull/6269/commits/0d5a33887f06ad77aa8f607f51da628f6c7056e8)

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**